src/EnvironmentCheckSuiteResult.php

```diff
@@ -35,12 +35,13 @@ public function __construct()
      * @param string $message
      * @param string $checkIdentifier
      */
-    public function addResult($status, $message, $checkIdentifier)
+    public function addResult($status, $message, $checkIdentifier, $responseTime)
```

Comment:
Can this be changed to default to null or '' to keep the API backwards compatible. Also phpdoc will need updating

Reply:
That won't keep it backwards compatible for anyone subclassing this class - the change in method signature will require a change in their subclass.

Reply:
My mistake - have updated to set a default null value and added the param to the phpdoc.

What might be the best way to preserve backwards compatibility for subclassing? I can only think of using `func_num_args()` and `func_get_arg()` which feels like overkill

Reply:
This is still not backwards compatible - when you subclass a class and override a method, you must match the method signature exactly. Therefore adding a new parameter _even if it has a default value_ is an API breaking change.